Sources/Penny/Handlers/InteractionHandler.swift

```diff
@@ -498,7 +498,10 @@ private extension InteractionHandler {
 
         switch subcommand {
         case GitHubSubCommand.link:
-            return "This command is still a WIP. Linking discordId: \(discordId) with GitHub account: \(String(describing: githubUsername))"
+            let clientID = try Constants.ghOAuthClientId.requireValue()
```

Comment:
just force-unwrap this. This a necessary environment value ... not-force-unwraping this necessary values will mostly just make the coding hard for us, than to make the code any more safe than it would be without the force unwrap.